manticore/manticore.py

```diff
@@ -52,6 +52,45 @@ def _check_disassembler_present(disasm):
     initial_state = State(constraints, platform)
     return initial_state
 
+def make_evm(args):
```

Comment:
what is the fate of this code? I like for now, we shouldn't try to devise some kind of disk evm binary format, and have evm features be API only. and from what this code looks like, the evm binary format would consist of pseudo-python anyway.

Reply:
Maybe this should load the json generated by something like:

```solc --combined-json abi,asm,ast,bin,bin-runtime,clone-bin,compact-format,devdoc,hashes,interface,metadata,opcodes,srcmap,srcmap-runtime,userdoc simple```

and perform a basic symbolic exploration on it? We can remove for now.

Reply:
i like that idea, but i think we should remove for now.